sections/all_map_locations.tex

```diff
@@ -3,18 +3,19 @@
 
 \begin{multicols*}{2}
 
-\hypertarget{All}{This} section describes the function of every single Field in the game.
-Any Fields not shown below or in the above Expansion section are explained by the reference sheet found at the back of the official rule book, pictured on the right. All such fields are \textbf{Visitable}.\par
+\hypertarget{All}{This} section describes the function of Fields not explained above.
+The following \textbf{Visitable} fields are not pictured out of redundancy, as their effects are described clearly by the symbols at the back of the official rule book (pictured on the right): \textbf{Water Wheel}, \textbf{Windmill}, \textbf{Mystical Garden}, \textbf{Shrine of Magic Gesture/Incantation}, \textbf{Learning Stone}, \textbf{Temple}, \textbf{Warrior's Tomb}, \textbf{Fountain of Youth}, \textbf{Treasure Symbol}, \textbf{Artifact Symbol}, \textbf{Resource Symbol}, \textbf{Pandora's Box}.\par
```

Comment:
Firstly, I suggest simply mentioning the table on the right (_...as their effects are described by the symbols in the table on the right, and at the back of the official rule book_). I reckon it will be easier for the readers to refer to the table, rather than the original, while reading this 😄

Secondly, that much bold text is rather hard to read. When everything's emphasised, then nothing's actually emphasised. I suggest making it a good old `itemize` in alphabetical order. Don't worry about the ballista.